test/units/modules/network/eos/test_eos_vlans.py

```diff
@@ -102,12 +102,12 @@ def test_eos_vlan_replaced(self):
         self.execute_show_command.return_value = []
         set_module_args(dict(
             config=[dict(
-                vlan_id=30,
-                name="thirty",
+                vlan_id=10,
+                name="tenreplaced",
                 state="suspend"
             )], state="replaced"
         ))
-        commands = ['vlan 30', 'name thirty', 'state suspend']
+        commands = ['vlan 10', 'no name', 'name tenreplaced', 'state suspend']
```

Comment:
This looks like the thing that it isn't supposed to be doing? I would expect the following instead:

```suggestion
        commands = ['vlan 10', 'name tenreplaced', 'state suspend']
```

This makes me think that `to_remove` is getting improperly set for name on a change instead of just absence

Reply:
Fixed this.